spec-0000/schedule.md

```diff
@@ -1,9 +1,3 @@
-2023 – Quarter 2:
```

Comment:
I would feel that keeping the last 1 or 2 Quarters in the list is more valueable information that the details of the future (in practice it's difficult to see anyone would consult this list to see what to drop a year ahead, as oppose to check what they should have dropped already)